### PR TITLE
feat(Rows): add icon button as control

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1722)

### CODE REVIEW

File: src/__stories__/list-story.tsx [hunk 1]
```diff
@@ -147,6 +150,27 @@ const Template: StoryComponent<Args & {boxed?: boolean}> = ({
...
+                        onPress: () => console.log('Pressed button'),
+                        type: 'brand',
+                        backgroundType: 'solid',
+                    },
+                };
+                break;
+            case 'iconButton and onPress':
+                controlProps = {
+                    iconButton: {
+                        Icon: IconPlayFilled,
```
Comment (yceballost) [inline src/__stories__/list-story.tsx:166]:
```suggestion
                        Icon: IconTrashCanRegular,
```

To let the play/pause icon for toggle example

File: src/__stories__/list-story.tsx [hunk 2]
```diff
@@ -147,6 +150,27 @@ const Template: StoryComponent<Args & {boxed?: boolean}> = ({
                     onPress,
                 };
                 break;
+            case 'iconButton':
+                controlProps = {
+                    iconButton: {
+                        Icon: IconPlayFilled,
```
Comment (yceballost) [inline src/__stories__/list-story.tsx:156]:
```suggestion
                        Icon: IconTrashCanRegular,
```

**Review by yceballost — CHANGES_REQUESTED** [review]

File: src/__stories__/list-story.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (marcoskolodny) [reply src/__stories__/list-story.tsx:156]: discussed offline. No need to make this story more complex. Added `small: false` to make it clear that you can customize it

File: src/__stories__/list-story.tsx
[the remark this answers is not in the document]
Comment (marcoskolodny) [reply src/__stories__/list-story.tsx:1]: Added

**Review by yceballost — APPROVED** [review]

File: src/icon-button.tsx [hunk 3]
```diff
@@ -156,25 +156,29 @@ export const RawDeprecatedIconButton = React.forwardRef<TouchableElement, Deprec
...
+    backgroundType?: IconButtonBackgroundType;
     bleedLeft?: boolean;
     bleedRight?: boolean;
     bleedY?: boolean;
 }
 
-type Props = BaseProps & ExclusifyUnion<HrefProps | ToProps | OnPressProps | MaybeProps>;
+export type IconButtonProps = BaseProps &
+    ExclusifyUnion<HrefProps | ToProps | OnPressProps | MaybeProps> &
+    ExclusifyUnion<{'aria-label': string} | {'aria-labelledby': string}>;
```
Comment (marcoskolodny) [inline src/icon-button.tsx:179]: forcing aria-label or aria-labelledby

**Comment by marcoskolodny** [comment]
> once [this one](https://github.com/Telefonica/mistica-web/pull/1034) is merged, should we support `ToggleIconButton` in `Rows` too?

@atabel yes, but that can be a second iteration (in order to avoid blocking this PR)

File: src/icon-button.tsx (hunk 3 above)
Comment (pladaria) [reply src/icon-button.tsx:179]: this is a breaking change

**Review by pladaria — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Pull request merged**